### ISSUE TITLE
add build script

### ISSUE THREAD

**Issue by wootw0rm** [Ghosts/Maus#2]
have idea to add build script for easy compile ?

**Comment by Rammelkast** [comment]
+1, Maven maybe?

**Comment by shetouane** [comment]
freezes every time i try to build a client

**Comment by Ghosts** [comment]
Looking into the best build solution currently, but if doing it yourself in IntelliJ you simply add a Jar build for the main Maus class. You do not need to build the Client jar.

@F17mous what OS are you on?

**Comment by Ghosts** [comment]
@wootw0rm @Rammelkast I've pushed an updated adding Maven Lifecycle support for "Package".
The output should be in /target/*.jar.

I've tested it on my machine and it all seems to be working correctly, however I'd love further testing & feedback.

I'm planning on focusing my development on stability, documentation, and cleaning up / commenting code to be a bit more open-source friendly. 

The next feature-release will hopefully be a remote desktop view, once I figure out the best approach.
After that I plan on introducing encrypted traffic (either obfuscated or RSA styled). 

EDIT:
Closing issue for now, if the Maven build has issues we'll consider that a separate issue. Thanks!
